src/app/helpers/mediaQueries.js

```diff
@@ -0,0 +1,15 @@
+import {
+  smartPhoneScreenWidthMin,
```

Comment:
I'd suggest renaming these to use `s`, `m`, `l`, `xl`, `xxl` since these are used by GEL Grid and familiar with teams across the BBC who are either using GEL Grid directly, or are using Grandstand, which pulls in GEL Grid. https://github.com/bbc/gel-grid#breakpoints

Reply:
We should discuss this.

Reply:
Lengthy comment added following our conversation.

Reply:
Where is the comment? Would like read - fear it might be lost.